### ISSUE TITLE
[Meds / Improvements] Implement revised content for CHAMPVA beneficiaries and others who use Meds by Mail

### ISSUE THREAD

**Issue by michaelbrodsky-a6** [department-of-veterans-affairs/va.gov-team#115450]
## Context
This ticket was flagged for Medications on July 28, 2025 in this [DSVA Slack thread](https://dsva.slack.com/archives/C04PRFEJQTY/p1753725712121239) from Robyn. 

## Background
This is a follow-up implementation ticket that emerged from this previous CAIA ticket ([#67895](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67895#issuecomment-3128171107)), which was created in 2023 but recently resurfaced. This CAIA ticket still has significant context. 

## Description
Implement **conditional** revised content for CHAMPVA beneficiaries and others who use Meds by Mail on the Medications landing page. This includes both **(a)** removing a sentence of copy from the `va.gov/my-health/medications` page; and **(b)** creating new content that is ever-present (not dynamically shown) for Meds by Mail users click.

## Details
- Remove this sentence from the top of the Medications page (at `va.gov/my-health/medications`): "If you use Meds by Mail, you can also call your servicing center and ask them to update your records."
- Create another content area including semantic H2 (but styled to look like H3, sorry! except on mobile, should be mobile h2), paragraph text, and an add'l information component <details><img width="1606" height="855" alt="Image" src="https://github.com/user-attachments/assets/4a388550-7398-4fdf-b27c-daff2f92b064" /></details>

### Components used
_Additional info component_ on Meds landing page `/my-health/medications`

#### H2 (looks like H3) / on mobile it's a mobile h2 semantically and stylistically
If you use Meds by Mail
#### Paragraph
We may not have your allergy records in our My HealtheVet tools. But the Meds by Mail servicing center keeps a record of your allergies and reactions to medications. 
#### Add'l info component header
How to update your allergies and reactions if you use Meds by Mail
#### Add'l info component dropdown content
If you have a new allergy or reaction, tell your provider. Or you can call us at 866-229-7389 or 888-385-0235 (TTY:711) and ask us to update your records. We're here Monday through Friday, 8:00 a.m. to 7:30 p.m. ET.

## Notes
- Figma Designs: [linked](https://www.figma.com/design/cqiuYyA1vn728D0Nde2NgS/Medications---Milestone-1---2?node-id=19648-38904&version-id=2257008410733159440&focus-id=19718-46104&m=dev)
- Be sure that the phone numbers are linked correctly from the alert copy
- The best way to identify Meds by Mail users is by searching for the "741MM" facility in user profiles
- The new content and add'l info component should be located on the "Medications" landing page (aka the Medications list view)

## Possible Tasks
- Finalize [list of questions](https://github.com/department-of-veterans-affairs/va.gov-team/issues/115450#issuecomment-3133137789) for the CHAMPVA Rx SME (Co Lai / Co.Lai@va.gov)
- Review the mock-up of the content on the page

## Acceptance Criteria
- [x] The Medications page (`va.gov/my-health/medications`) has been updated with revised copy
- [x] QA Review
- [x] A11y Review

**Comment by michaelbrodsky-a6** [comment]
**List of Questions for the CHAMPVA Rx SME:**

- What's the best way to identify Meds by Mail users?
- Does the 741MM facility get removed from a user's profile if they are no longer a CHAMPVA beneficiary?

**Comment by michaelbrodsky-a6** [comment]
**Notes from Backlog Refinement on July 30, 2025:** 

**(a)** Sara mentioned that the best way to identify Meds by Mail users is by using the "741MM facility" in their user profiles.

**(b)** @michaelbrodsky-a6 will locate the url that the proposed "Find your Meds by Mail servicing center" CTA should link to, and add it to the body copy of this ticket (outlined above).

**(c)** @sterkenburgsara will create a mock that indicates how the informational alert should look / where on the page it should live. Then @michaelbrodsky-a6 will add it to this ticket.

**(d)** Once the to-do's outlined above are completed, this ticket will be ready to be moved from the Icebox to the Backlog.
